site/src/routes/docs/_sections.js

```diff
@@ -40,10 +40,20 @@ const blockTypes = [
 export default function() {
 	const makeSlug = makeSessionSlugProcessor(SLUG_PRESERVE_UNICODE);
 
+	const makeSectionSlug = filename => {
+		// '01-component-format' -> 'component format'
+		const phrase = filename.replace(/^\d+-/, '').replace(/-/, ' ');
+		const capitalised = phrase.substring(0, 1).toUpperCase() + phrase.substring(1);
+
+		return makeSlug(capitalised);
+	};
+
 	return fs
 		.readdirSync(`content/docs`)
 		.filter(file => file[0] !== '.' && path.extname(file) === '.md')
 		.map(file => {
+			const sectionSlug = makeSectionSlug(path.parse(file).name);
```

Comment:
We want to base the slug on the `title` metadata from the markdown file (which is what's rendered in the nav bar), and not on the filename.

Reply:
you're right, fixed!